### PR TITLE
Make `IsBlank` operate on `Text`

### PR DESCRIPTION
**Pull request by andreoss**
- Make `IsBlank` match `Text` object

### CODE REVIEW

**Comment by victornoel** [comment]
@0crat status

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/llorllale/cactoos-matchers/issues/247#issuecomment-858361392))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#247](https://github.com/llorllale/cactoos-matchers/issues/247) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Allorllale%2Fcactoos-matchers%23247%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/llorllale/cactoos-matchers/issues/247#issuecomment-864413723))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#247](https://github.com/llorllale/cactoos-matchers/issues/247) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Allorllale%2Fcactoos-matchers%23247%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by victornoel** [comment]
@0crat in

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) everybody who has role `REV` is banned at [#247](https://github.com/llorllale/cactoos-matchers/issues/247); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) everybody who has role `REV` is banned at [#247](https://github.com/llorllale/cactoos-matchers/issues/247); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: src/main/java/org/llorllale/cactoos/matchers/IsBlank.java [hunk 1]
```diff
@@ -26,22 +26,27 @@
...
-     */
-    public IsBlank() {
-        super(
-            new MatcherOf<>(
-                text -> text.trim().isEmpty(),
-                desc -> desc.appendText("is blank"),
-                (text, desc) -> desc.appendText("was ").appendValue(text)
-            )
-        );
+public final class IsBlank extends TypeSafeMatcher<Text> {
```
Comment (victornoel) [inline src/main/java/org/llorllale/cactoos/matchers/IsBlank.java:40]: @andreoss this should be based on `TextMatcher` like other `Matcher` on `Text`.

**Review by victornoel — COMMENTED** [review]
@andreoss thx, here is a review

File: src/main/java/org/llorllale/cactoos/matchers/IsBlank.java (hunk 1 above)
Comment (andreoss) [reply src/main/java/org/llorllale/cactoos/matchers/IsBlank.java:40]: @victornoel Done

**Comment by victornoel** [comment]
@rultor merge

**Pull request merged**